components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -188,7 +214,52 @@ private IdentityRecoveryConstants() {
 
         // UAV - User Account Verification.
         ERROR_CODE_UNSUPPORTED_VERIFICATION_CHANNEL("UAV-10001",
-                "Unsupported verification channel");
+                "Unsupported verification channel"),
+
+        // UNR - Username Recovery
+        ERROR_CODE_USERNAME_RECOVERY_NOT_ENABLED("UNR-10001", "Username recovery is not enabled"),
```

Comment:
let's use the approach finalized in [1] for error codes
[1] "[Architecture] WSO2 Identity Server REST API Error Response Standardization"

Reply:
Since this is a User API, error codes follow the API Error Response Standardization.